docs/security-policy.md

```diff
@@ -39,16 +39,25 @@ spec:
     - vmSelector:
         matchLabels:
           role: db
+        matchExpressions:
+          - {key: k1, operator: In, values: [a1,a2]}
+          - {key: k2, operator: NotIn, values: [a3,a4]}
+ 
   rules:
     - direction: in
       action: allow
       sources:
         - namespaceSelector:
             matchLabels:
               role: control
+            matchExpressions:
+              - {key: k3, operator: Exists}
+
         - podSelector:
             matchLabels:
               role: frontend
+            - {key: k4, operator: DoesNotExist}
```

Comment:
The syntax looks not correct. Could you verify it before editing?

Reply:
Thanks for correction. It's has been fixed and verified.